src/api/app/assets/stylesheets/webui/application/style.scss

```diff
@@ -412,3 +412,10 @@ div.dataTables_wrapper:after {
 #subheader #breadcrump {
   line-height: 1.3em;
 }
+
+// flash messages
+.flash-message {
+  p {
+      margin: 10px;
+  }
+}
```

Comment:
Wasn't this styled before? btw this should have been in a separate commit with an explanation why this is needed in the commit message.